dfx.json

```diff
@@ -1,6 +1,28 @@
 {
     "canisters": {
-        "backend": {
+       "backend": {
+         "build": "npx azle backend",
+         "candid": "src/js_backend/js_backend.did",
+         "candid_gen": "custom",
+         "declarations": {
+           "node_compatibility": true
+         },
+         "gzip": true,
+         "main": "src/js_backend/src/index.ts",
+         "type": "custom",
+           "wasm": ".azle/backend/backend.wasm",
+          "metadata": [
+                {
+                    "name": "candid:service",
+                    "path": "src/js_backend/js_backend.did"
+                },
+                {
+                    "name": "cdk:name",
+                    "content": "azle"
+                }
+            ]
+        },
+        "mo_backend": {
             "type": "motoko",
             "main": "src/backend/AuctionServer.mo"
         },
```

Comment:
This could be removed or we could toggle between backend by renaming one canister to backend and the other to something else.